infrastructure/main.tf

```diff
@@ -21,6 +27,12 @@ locals {
   vault_name                = "${var.product}si-${var.env}"
 }
 
+data "azurerm_subnet" "postgres" {
+  name                 = "recipe-postgres"
```

Comment:
you shouldn't have an application subnet name,
you probably don't want to call it postgres as other private endpoints could use it.

in SDS we just use one called 'iaas', (known by people around at the time as the 'random iaas' subnet)